pkg/controller/servicebindingrequest/controller.go

```diff
@@ -36,70 +34,58 @@ func newReconciler(mgr manager.Manager) (reconcile.Reconciler, error) {
 	return &Reconciler{client: mgr.GetClient(), dynClient: dynClient, scheme: mgr.GetScheme()}, nil
 }
 
-func addServiceBindingRequestWatches(c controller.Controller) error {
+// add adds a new Controller to mgr with r as the reconcile.Reconciler.
+func add(mgr manager.Manager, r reconcile.Reconciler) error {
+	opts := controller.Options{Reconciler: r}
+	c, err := controller.New("servicebindingrequest-controller", mgr, opts)
+	if err != nil {
+		return err
+	}
+
 	pred := predicate.Funcs{
 		UpdateFunc: func(e event.UpdateEvent) bool {
-			data, err := runtime.DefaultUnstructuredConverter.ToUnstructured(e.ObjectNew)
-			if err != nil {
-				// TODO: add logging to show this error;
-				return false
-			}
-
-			u := &unstructured.Unstructured{Object: data}
-			sbr := &v1alpha1.ServiceBindingRequest{}
-			err = runtime.DefaultUnstructuredConverter.FromUnstructured(u.Object, sbr)
-			if err != nil {
-				// TODO: add logging to show this error;
-				return false
-			}
+			/*
+				sbrSelector, err := GetSBRNamespacedNameFromObject(e.ObjectNew)
+				if err != nil {
+					// TODO: add logging to show this error;
+					return false
+				}
+				if IsSBRNamespacedNameEmpty(sbrSelector) {
+					return false
+				}
+			*/
 
-			// allowing pending SBR to be reconciled anyways
-			// FIXME: use the constant defined in catchall reconciler;
-			if sbr.Status.BindingStatus == "pending" {
-				return true
-			}
-
-			// Ignore updates to CR status in which case metadata.Generation does not change
+			// ignore updates to CR status in which case metadata.Generation does not change
 			return e.MetaOld.GetGeneration() != e.MetaNew.GetGeneration()
 		},
 		DeleteFunc: func(e event.DeleteEvent) bool {
-			// Evaluates to false if the object has been confirmed deleted.
+			// evaluates to false if the object has been confirmed deleted
```

Comment:
Do we have any guidelines for code comments?

Reply:
Simple answer, no.